lib/BackgroundJob/Tasks/AddMissingImagesTask.php

```diff
@@ -164,13 +163,7 @@ private function parseUserFolder(int $model, Folder $folder): int {
 	 * @return array List of all images and folders to continue recursive crawling
 	 */
 	private function getPicturesFromFolder(Folder $folder, $results = array()) {
-		// todo: should we also care about this too: instanceOfStorage(ISharedStorage::class);
```

Comment:
Easy.. No?
I just ignore any control and accept all the files. 

Well, If we want to make it more serious, we have a problem, if you print the class of the shared folder, does this say TRASH ??? (We had already seen this :confused:)

But in all the implementations that I saw, (RichDocuments, Gallery, Core) do not control the type of storage .. Just if is local..